### DISCUSSION TITLE
Why is the name KaTeX and not JaTeX?

### DISCUSSION THREAD

**Discussion by hakanai** [KaTeX/KaTeX#3322] [General] [answered]
I'm a Kotlin developer and was considering writing a Kotlin port of TeX, but lo and behold the most sensible name for the project was already taken. But it was taken by a JavaScript library, so why was it KaTeX and not JaTeX?

I'm not complaining, just curious. I figure there must be some logic to it but I don't follow JavaScript at all so I don't have any idea how it relates.

**Comment by JacobBeaudry** [comment, marked as answer]
I believe it's in reference to the Khan Academy, check out the [wiki](https://en.wikipedia.org/wiki/KaTeX).
